packages/types/src/deneb/types.ts

```diff
@@ -48,4 +49,4 @@ export type LightClientOptimisticUpdate = ValueOf<typeof ssz.LightClientOptimist
 export type LightClientStore = ValueOf<typeof ssz.LightClientStore>;
 
 export type ProducedBlobSidecars = Omit<BlobSidecars, "signedBlockHeader" | "kzgCommitmentInclusionProof">;
-export type Contents = Omit<BlockContents, "block">;
+export type Contents = Omit<BlockContents<ForkName.deneb>, "block">;
```

Comment:
we can move this out to ../types as well, also don't need to zero down to deneb type

Reply:
That will be breaking change for `deneb` namespace. Want to avoid any such changes.

Reply:
its not breaking since there is just deneb in unstable right now, but ok if you wanna follow it up with another PR to not pile up in this PR